### ISSUE TITLE
Nested template elements break html beautify

### ISSUE THREAD

**Issue by patroza** [beautify-web/js-beautify#1032]
Input:

```
<template class="row">
  <div>
    <template>
    </template>
  </div>

  <div></div>
</template>
```

Observed output: (in >= 1.6.0)

```
<template class="row">
  <div>
    <template>
    </template>
</div>

<div></div>
</template>
```

Expected output: (like in 1.5.10)

```
<template class="row">
  <div>
    <template>
    </template>
  </div>

  <div></div>
</template>
```

Removing the `'template'` entry in beautify-html.js `unformatted` array, works around the issue

**Comment by atishpatel** [comment]
I believe my issue is related to this one. I'm using "wrap_attributes": "force-aligned" and it works great everywhere except inside a template tag.

input:
`
<div id="1" class="2" test="3"></div>
<template>
<div id="1" class="2" test="3"></div>
</template>
`
output:
`
<div id="1"
        class="2"
        test="3"></div>
<template>
    <div id="1" class="2" test="3"></div>
</template>
`

**Comment by bitwiseman** [comment]
The issue is related to #841.  Unformatted and inline are not distinguished in the html beautifier and they need to be.

**Comment by bitwiseman** [comment]
Fixed in 1.8.0-rc2
